cmd/contour/serve.go

```diff
@@ -652,32 +652,10 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 
 		switch ctx.Config.Server.XDSServerType {
```

Comment:
Non-blocking question: Is this the switch between Contour and go-control-plane? Sorry, I've missed some stuff here in the last little while.